esp_lcd_qemu_rgb/interface/esp_lcd_qemu_rgb.h

```diff
@@ -13,12 +13,17 @@
 extern "C" {
 #endif
 
+typedef enum {
+    BPP_16 = 16,
+    BPP_32 = 32
+} BppEnum;
```

Comment:
Can you rename this to have the same naming convention as other IDF structures? Something like `esp_lcd_rgb_qemu_bpp_t`